src/meta/processors/admin/Balancer.cpp

```diff
@@ -368,8 +389,8 @@ cpp2::ErrorCode Balancer::leaderBalance() {
     std::vector<GraphSpaceID> spaces;
     kvstore::ResultCode ret = kvstore::ResultCode::SUCCEEDED;
     if (!getAllSpaces(spaces, ret)) {
-        LOG(ERROR) << "Can't access kvstore, ret = d"
-                   << static_cast<int32_t>(ret);
+        inLeaderBalance_ = false;
```

Comment:
inLeaderBalance_ is not set to true, why need set to false?

Reply:
It is set in CAS below. It is set to false in getAllSpaces, and I move this line here because `getAllSpaces` is used by balance data and balance leader.

Reply:
这样修改可能有点问题，看leaderBalance函数的逻辑，inLeaderBalance_变量还有多线程同步的作用，如果多个线程进入leaderBalance函数，该变量保证只有一个线程会进入client_->transLeader流程，假设thread a已经进入client_->transLeader流程，如果第二个线程进入了leaderBalance函数，但是getAllSpaces失败，这样就把inLeaderBalance_设置为false，此时第三个线程进入这个函数的时候，就会和第一个一起进入client_->transLeader流程

Reply:
👍 My bad